config/base.yml

```diff
@@ -923,6 +923,9 @@ Performance/CollectionLiteralInLoop:
 Performance/CompareWithBlock:
   Enabled: true
 
+Performance/ConcurrentMonotonicTime:
+  Enabled: true
```

Comment:
This cop does not support Rubies older than 2.3 (based on [this issue linked from the rubocop PR](https://github.com/rails/rails/pull/43502)), so to enable it in the base means it should be disabled in the 2.2 config file.